### DISCUSSION TITLE
Watch an input value when using use-form

### DISCUSSION THREAD

**Discussion by mrdeeds88** [mantinedev/mantine#5318] [Q&A] [answered]
I wonder is there any way we can watch an input value with use-form just like "watch" from react-hook-form.
Thanks!

**Comment by rtivital** [comment, marked as answer]
`useEffect(() => {}, [form.values.yourValue])`

**Reply by mrdeeds88** [reply]
Many thanks, It works.  I have never check for this as I thought form.values only return current values on submitting.
